.vib/rabbitmq/cypress/cypress/integration/rabbitmq_spec.js

```diff
@@ -0,0 +1,100 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  const APP_NAME = 'RabbitMQ';
+
```

Comment:
I'm still not sure if this should be a CONST or a fixture.
I think it's a CONST since it's metadata (application name), not the actual test data we are adding.
What do the reviewers think?

Reply:
I would say that using a `const` for this matches the other instances where we used `const`, though we may be misusing them. Maybe we should be opting to add the const value directly to the tests.

Reply:
Regarding the value of the test, I wanted to anchor to a page element just to check if the login was successful. 
![RabbitMQ](https://user-images.githubusercontent.com/47592680/168294094-5bbaa96e-ae44-40e1-b704-8e5bca14351d.png)

I'm not testing this element as such.  I can apply the Regex improvement, I think it makes sense. 
I will leave the value directly at the test then.
I'm still not really clear on fixture/const/direct usage. We can brainstorm next week!

Reply:
Applied the improvement!